service/src/test/java/bio/terra/tanagra/service/VumcAdminServiceTest.java

```diff
@@ -24,12 +25,14 @@ public class VumcAdminServiceTest {
   @Autowired private VumcAdminService vumcAdminService;
 
   @Test
+  @Disabled
```

Comment:
I think these are the only two tests in this class -- perhaps move up the `@Disabled` annotation up to the class level.